src/DynamoRevit/DynamoRevitApp.cs

```diff
@@ -254,6 +323,12 @@ public static Assembly ResolveAssembly(object sender, ResolveEventArgs args)
 
             try
             {
+                assemblyPath = Path.Combine(DynamoRevitApp.DynamoCorePath, assemblyName);
```

Comment:
DynamoCorePath will be computed properly based on you assembly path and it can consider your debug binaries appropriately. So once I submit this PR, your current setup will continue to work.